### PR TITLE
Fix dependencies & rename

### PR DESCRIPTION
**Pull request by gluck**

### CODE REVIEW

File: alert/doc/doc.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ```js script
-import { html, registerIconLibrary } from '~/md-layout';
+import { html, registerIconLibrary } from '@divriots/starter-kicks/md-layout';
```
Comment (gqio) [inline alert/doc/doc.md]: Why we should not use anymore the `~` ?

Comment (bashmish) [reply alert/doc/doc.md]: @gqio https://github.com/divriots/webcomponents-editor/pull/2556

Comment (gluck) [reply alert/doc/doc.md]: I updated all paths in md/mdx/stories without more thoughts. Ideally should be replaced only when visible in the docs, but that's quite the burden to check everywhere whether it's visible or not.
(but for this starter it probably makes no sense actually)

Comment (gqio) [reply alert/doc/doc.md]: ok it's a major change this one I missed it

Comment (bashmish) [reply alert/doc/doc.md]: actually for `md-layout` it doesn't make sense, I'd revert this change

otherwise LGTM!

**Pull request merged**